src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,345 @@
+===================
+Writing a narrative
+===================
+
+The following will walk you through writing a Nextstrain narrative.
+We'll start with a basic 2-slide narrative and then gradually add further slides with the intention
+of describing the functionality of narratives. The aim is for you to feel comfortable with
+the technical aspects of creating and troubleshooting narratives.
+
+
+For an introduction to the concept of Narratives, please see `Communicating Results Using Narratives </guides/communicate/narratives-intro.html>`__ or take a look at this example narrative,  `Twenty years of West Nile virus <https://next.nextstrain.org/narratives/twenty-years-of-WNV>`__.
+
+This tutorial will rely on the new `Narratives Debugger <https://dev.nextstrain.org/edit/narratives>`__ which allows you to drag & drop the markdown files we'll introduce here and see an instant summary of the narrative.
+
+.. TODO- update links above once functionality in nextstrain.org
+
+.. contents:: Sections in this document
+  :local:
+  :depth: 2
+
+.. note::
+
+  This guide uses the narratives debugger on nextstrain.org; while you can also use this by running Auspice on your computer there are some differences - see :ref:`below for more details <auspice_vs_nextstrain>`.
+
+
+Prerequisites
+=============
+
+In order to make sense of this tutorial we are going to need:
+
+* A (basic) understanding of writing markdown `(here is good introduction) <https://learnxinyminutes.com/docs/markdown/>`__.
+* Familiarity with the general concept of Nextstrain Narratives (see `Communicating Results Using Narratives </guides/communicate/narratives-intro.html>`__ to get started).
```

Comment:
This sort of link should be `:doc:` references, e.g.

```suggestion
* Familiarity with the general concept of Nextstrain Narratives (see :doc:`/guides/communicate/narratives-intro` to get started).
```

You can omit the link text if you want to use the document's title.

There's several more examples of this mistake throughout the document.

Note also that while links like `/guides/communicate/narratives-intro.html` will appear to work in the review site, they won't on the live site.

Reply:
> Note also that while links like /guides/communicate/narratives-intro.html will appear to work in the review site, they won't on the live site.

Ouch! That's quite the papercut. Changing them now...

Reply:
Fixed up - thanks!